src/main/CMV.java

```diff
@@ -94,7 +94,54 @@ public boolean lic5_calculate() {
         return false;
     }
 
-    private Boolean lic6_calculate() {
+    public Boolean lic6_calculate() {
+        int N_PTS = Parameters.N_PTS;
+        double DIST = Parameters.DIST;
+        if(N_PTS < 3 || N_PTS > datapoints.length || DIST <= 0)
+            return false;
+        
+        // Go through each set of N_PTS consecutive points
+        for (int point = 0; point < datapoints.length; point++) {
+            // First and last point within the set of consecutive points
+            int p1 = point;
+            int p2 = (point + N_PTS - 1) % datapoints.length;
```

Comment:
unsure wheter p2 > datapoints.length should loop back to the start. Don't know if that is really considered _consecutive_.

Reply:
I agree with your view on consecutive order. Will change that.